kitchen/wazuh-puppet/kitchen/test/base/test_wazuh_agent.py

```diff
@@ -14,15 +14,13 @@ def test_wazuh_agent_package(host,get_wazuh_version):
     assert pkg.is_installed
     assert pkg.version.startswith(version)
 
-
-@pytest.mark.filterwarnings('ignore')
 @pytest.mark.skipif('manager' in os.environ.get('KITCHEN_INSTANCE'), reason='Skip on wazuh manager instances')
-@pytest.mark.parametrize("wazuh_service, wazuh_owner", (
```

Comment:
Why removing this? We can test both `agent.is_running` and then ensure all processes are running

Reply:
Hi @jm404 !

Totally agree with you.

I Added the daemons' processes tests for both `wazuh-manager` and `wazuh-agent`.

Kr,

Rshad